pkgs/misc/themes/sddm/chili/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, qtbase, qtquickcontrols, qtgraphicaleffects }:
+
+/*
+  This derivation installs the chili theme. Use
+    `services.xserverdisplayManager.sddm.theme = "chili";`
+  to enable the theme.
+ */
+
+stdenv.mkDerivation rec {
+  pname = "chili-sddm-theme";
+  version = "0.1.5";
+  
+  src = fetchFromGitHub {
+    owner = "MarianArlt";
+    repo = "sddm-chili";
+    rev = "${version}";
+    sha256 = "036fxsa7m8ymmp3p40z671z163y6fcsa9a641lrxdrw225ssq5f3";
+  };
+
+  buildInputs = [ qtbase qtquickcontrols qtgraphicaleffects ];
```

Comment:
These don't actually seem to do anything - afaict there is no build for this derivation?

Reply:
Yes, just an unzip. What is the recommended phase to add the qt deps?

Reply:
Well, where are they needed? Presumably sddm uses them when it loads the theme. If this works (which I guess it does if you tested it?) then presumably sddm already has the dependencies around when it loads it. In which case you can delete these.

Reply:
I tested it. Without this line it does not work. I guess some qt path are set automatically.

Reply:
Hm indeed, somehow something is putting them in propagated build inputs. I always find that a bit magical, but it seems to work.